app/controllers/application_controller.rb

```diff
@@ -1,2 +1,25 @@
+# frozen_string_literal: true
+
+# ApplicationController
 class ApplicationController < ActionController::Base
+  helper_method :current_user
```

Comment:
Please move all helper methods in helper controllers. Helper methods should provide results that will be used only in views.